### ISSUE TITLE
Cockatrices Not Following Me

### ISSUE THREAD

**Issue by CommandBlockGuru** [Alex-the-666/Ice_and_Fire#2619]
The cockatrice has to be my favorite animal in this mod, #1. I'm obsessed with them. In my 1.12.2 modded survival world, I even tried to make an army, with the roosters guarding my house, and the hens following me if I ever had to travel. But none of my cockatrices are following me when I tell them to. I don't know if this is a bug or it's because I have too many spawned. Another weird thing is that they only stop following me after I leave the game. Before I leave, if I'd just spawned them, they follow me, but after, they just stand there. I love this mod and I really want this bug to be fixed so I can continue to grow my army. Thank you!

**Comment by CommandBlockGuru** [comment]
Ok, so I figured out a kinda solution- I just use the golden lassos from the tinkerer something mod and when I put em back down they follow me.

**Comment by darchitectlp** [comment]
Also having this issue. I set the cockatrice to follow, and it won't. If I right click with an empty hand, garbled text shows on the screen. By quickly right clicking, I can make out one line that says "This cockatrice's home position was removed" and then there's another line that shows up immediately on top of it... Some thing about "Home position set" and some coordinates. Any chance this could be fixed? Version 1.9.1. Thanks!
